src/libstd/fs.rs

```diff
@@ -2261,11 +2282,41 @@ mod tests {
     }
 
     #[test]
+    fn concurrent_recursive_mkdir() {
+        for _ in 0..50 {
```

Comment:
Does this test really need to be run 50 times? It fails the first time for me on Windows.